cypress/e2e/testTrello.cy.js

```diff
@@ -5,13 +5,19 @@ describe('Boards', function(){
     before(()=> {
         cy.login();
     });
-    it('Create a new board', () => {
-        cy.get('li[data-testid="home-team-tab-section-6442879b62c449644bea42b0"] span[data-testid="DownIcon"]').click();
-        cy.get('[data-testid="home-team-boards-tab"] > .DD3DlImSMT6fgc').click();
-        cy.get('.QB_5E6Ho6209bY > .bxgKMAm3lq5BpA').click();
-        cy.get('[data-testid="create-board-title-input"]').type('Test 01');
-        cy.get('[data-testid="create-board-submit-button"]').click();
-        //assertion if "Test 01" was created
+    it('Create new board, modify board title, and delete board', () => {
+        //creating a board and assertion if it was created.
+        cy.createBoard();
         cy.get('[class="js-board-editing-target board-header-btn-text"]').should('have.text', 'Test 01');
+        //Modify board title
+        cy.get('[class="js-board-editing-target board-header-btn-text"]').type('Test 02{enter}');
+        cy.get('[class="js-board-editing-target board-header-btn-text"]').should('have.text', 'Test 02');
+        //Delete the board
+        cy.deleteBoard();
+        cy.contains('Test 02').should('not.exist');
```

Comment:
The delete actions should be in an after or afterEach hooks, please review which one.

Reply:
Split your test into two tests or it.
One for create, one for Edit.
And for each one the created Board should be deleted in the After Hook, not in the test body.